src/controller/index.ts

```diff
@@ -214,6 +218,27 @@ export default class SimulariumController {
         }
     }
 
+    public convertAndLoadTrajectory(
+        netConnectionConfig: NetConnectionParams,
+        obj: Record<string, unknown>,
```

Comment:
in addition to typing this, I would call it something clearer, like `dataToConvert` or `trajectoryPayload` (open to suggestions :)